dashboard/dashapi/dashapi.go

```diff
@@ -393,6 +397,7 @@ type BugNotification struct {
 	Text        string   // meaning depends on Type
 	CC          []string // additional CC emails
 	Maintainers []string
+	Recipients  Recipients
```

Comment:
Does this replace Maintainers and Cc?  If so, can you leave a comment noting that Maintainers and Cc are deprecated, similar to the other cases?